materialrangebar/src/main/res/values/attrs.xml

```diff
@@ -24,6 +25,9 @@
         <attr name="mrb_pinMaxFont" format="dimension"/>
         <attr name="mrb_rangeBarPaddingBottom" format="dimension"/>
         <attr name="mrb_selectorColor" format="reference|color"/>
+        <attr name="mrb_selectorColorLeft" format="reference|color"/>
+        <attr name="mrb_selectorColorRight" format="reference|color"/>
```

Comment:
_Non-Blocking:_ @Rlshep What do you think about naming them better so that there is no vagueness e.g:
`mrb_selectorColorLeft` could be renamed to `mrb_leftSelectorColor`
`mrb_selectorColorRight ` could be renamed to `mrb_rightSelectorColor`

Reply:
Renamed left & right. is 1. if the number of ticks displayed is less than... still an issue? If so how are you recreating it?

Reply:
Just replace the `rangebar` xml in `activity_main` with this 
```
    <com.appyvet.materialrangebar.RangeBar
        xmlns:app="http://schemas.android.com/apk/res-auto"
        android:id="@+id/rangebar1"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_marginStart="40dp"
        android:layout_marginEnd="40dp"
        app:mrb_pinMaxFont="15sp"
        app:mrb_pinMinFont="12sp"
        app:mrb_rangeBarPaddingBottom="30dp"
        app:mrb_selectorBoundaryColor="@color/accent"
        app:mrb_selectorBoundarySize="2dp"
        app:mrb_pinTextColor="@color/white"
        app:mrb_temporaryPins="true"
        app:mrb_selectorSize="10dp"
        app:mrb_tickLabelColor="@color/indigo500"
        app:mrb_tickLabelSelectedColor="@color/accent"
        app:mrb_tickBottomLabels="@array/ticks_labels"
        app:mrb_tickTopLabels="@array/ticks_labels"
        app:mrb_tickDefaultLabel="label"
        app:mrb_tickLabelSize="4sp"
        app:mrb_tickEnd="10"
        app:mrb_tickInterval="1"
        app:mrb_tickStart="7"
        app:mrb_tickHeight="4dp"
        app:mrb_pinRadius="14dp"
        app:mrb_rangeBar_rounded="true"
        app:mrb_connectingLineColors="@array/connecting_colors"
        app:mrb_tickDefaultColor="@color/indigo500"
        app:mrb_tickColors="@array/tick_colors"
        app:mrb_rangeBar="true"
        app:mrb_pinColor="#6c3f6a"
        app:mrb_leftSelectorColor="#FF5722"
        app:mrb_rightSelectorColor="#E91E63"/>
```

Once you run the app and move the selectors you will see that last 2 colours are the same (In reality the last one should be green)


![Screenshot_1556095639](https://user-images.githubusercontent.com/2201511/56645640-5b964d00-667e-11e9-9b21-0c5005f97bbc.png)

Reply:
Fixed